native/explorer/src/datatypes/ex_dtypes.rs

```diff
@@ -108,13 +107,15 @@ impl TryFrom<&DataType> for ExSeriesDtype {
             )?))),
 
             DataType::Struct(fields) => {
-                let mut struct_fields = HashMap::new();
+                let mut struct_fields = Vec::new();
 
                 for field in fields {
                     struct_fields
-                        .insert(field.name().to_string(), Self::try_from(field.data_type())?);
+                        .push((field.name().to_string(), Self::try_from(field.data_type())?));
                 }
 
+                struct_fields.sort_by(|(a, _), (b, _)| a.cmp(b));
```

Comment:
Why are we sorting the struct fields here? Shouldn't we keep the order given by `fields`?

Reply:
I think this was not correct, so I removed.